_config.yml

```diff
@@ -7,6 +7,7 @@ syntax_highlight_theme: "tomorrow-night"
 cdnurl: "https://cdn.tiny.cloud/1/no-api-key/tinymce/5/tinymce.min.js"
 tinymce_live_demo_url: "https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/5/tinymce.min.js"
 tinydrive_live_demo_url: "https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinydrive/stable/tinydrive.min.js"
+webcomponent_url: "https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@0.1.0/dist/tinymce-webcomponent.min.js"
```

Comment:
That is a URL specifically for 0.1.0, it is about to be outdated (when I merge a PR)...

We gave up on the 0.x idea after it was apparent we could not unpublish the 1.x stuff so it is better to use the URL for the latest 1.x release:
https://cdn.jsdelivr.net/npm/@tinymce/tinymce-webcomponent@1/dist/tinymce-webcomponent.js

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1659/commits/59b9c66a45db648e5a9b718b6755ba390d539161